docs/ref/models/database-functions.txt

```diff
@@ -841,6 +841,34 @@ that deal with time-parts can be used with ``TimeField``:
 JSON Functions
 ==============
 
+``JSONArray``
+-------------
+
+.. versionadded:: 5.2
+
+.. class:: JSONArray(*expressions)
+
+Accepts a list of field names or expressions and returns a JSON array
+containing those values. SQL ``NULL`` will be mapped to JSON scalar
+``null`` on all supported backends.
```

Comment:
```suggestion
containing those values.
```
I am tempted to remove this note as I feel it's more confusing than helpful.
I feel this behavior is [documented here](https://docs.djangoproject.com/en/5.1/topics/db/queries/#storing-and-querying-for-none):
> This only applies to `None` as the top-level value of the field. If `None` is inside a [list](https://docs.python.org/3/library/stdtypes.html#list) or [dict](https://docs.python.org/3/library/stdtypes.html#dict), it will always be interpreted as JSON `null`.

So to me this is expected and we perhaps don't need to call it out?

Reply:
Yeah, I think that's probably the expected behavior given the other supporting documentation and behavior elsewhere.